### PR TITLE
Attempts to fix war1tool muxing on macOS

### PR DESCRIPTION
**Pull request by shinra-electric**

### CODE REVIEW

File: war1tool.cpp [hunk 1]
```diff
@@ -1926,8 +1926,8 @@ void MuxAllIntroVideos() {
 	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
 	    "%s/%s/title_s.avi"; // video filter crop not supported on windows
```
Comment (Jarod42) [inline war1tool.cpp:1927]:
```suggestion
	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
	    "\"%s/%s/title_s.avi\""; // video filter crop not supported on windows
```
Would also fix on windows for path with space.

File: war1tool.cpp [hunk 2]
```diff
@@ -1911,13 +1911,13 @@ void MuxAllIntroVideos() {
 	fprintf(mylist, "file '%s/%s/%s'\n", Dir, SOUND_PATH, "intro_4.wav");
 	#endif
 	fclose(mylist);
-	const char* cmd3a = "ffmpeg -y -f concat -safe 0 -i %s "
+	const char* cmd3a = "ffmpeg -y -f concat -safe 0 -i \"%s\" "
 		"%s/%s/cave_a.wav";
```
Comment (Jarod42) [inline war1tool.cpp:1915]:
```suggestion
		"\"%s/%s/cave_a.wav\"";
```
You miss this one.

**Comment by shinra-electric** [comment]
[Terminal Saved Output.txt](https://github.com/Wargus/war1gus/files/14547175/Terminal.Saved.Output.txt)

Hmmm....

File: war1tool.cpp [hunk 3]
```diff
@@ -1911,23 +1911,23 @@ void MuxAllIntroVideos() {
...
 	#ifdef WIN32
-	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
-	    "%s/%s/title_s.avi"; // video filter crop not supported on windows
+	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
+	    "\"%s/%s/title_s.avi\""; // video filter crop not supported on windows
 	#else
-	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
-		"-vf 'crop=640:288:0:0' %s/%s/title_s.avi";
+	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
+		"-vf 'crop=640:288:0:0'\" %s/%s/title_s.avi\" ";
```
Comment (Jarod42) [inline war1tool.cpp:1930]:
```suggestion
		"-vf 'crop=640:288:0:0' \"%s/%s/title_s.avi\" ";
```
Misplaced space.
Unsure why you add extra space at the end...

I created https://github.com/Wargus/war1gus/tree/quotes_fix
which is mostly a clean (from git history) work of your branch.

Comment (shinra-electric) [reply war1tool.cpp:1930]: Nice catch, thanks.

**Comment by shinra-electric** [comment]
I tested out the new branch, and it now muxes successfully. 

Thanks a lot for your patience.

**Comment by shinra-electric** [comment]
Closing in favour of https://github.com/Wargus/war1gus/tree/quotes_fix

**Pull request closed without merging**